site2/docs/reference-cli-tools.md

```diff
@@ -734,9 +734,9 @@ Options
 
 |Flag|Description|Default|
 |---|---|---|
-|`-c`, `--conf`|Configuration for the auto-recovery daemon||
+|`-c`, `--conf`|Configuration for the auto-recovery||
 |-readOnly|Force start a read-only bookie server|false|
-|-withAutoRecovery|Start auto-recovery service bookie server|false|
+|-withAutoRecovery|Start autorecovery service bookie server|false|
```

Comment:
Do you miss this one? (and many other places)

Reply:
Ok i will take a full check on it.